### DISCUSSION TITLE
在版本设置概览中添加自定义的文件夹快捷方式

### DISCUSSION THREAD

**Discussion by Raiden-Yayi** [Meloong-Git/PCL#1847] [提问] [answered]
可以自定义添加文件夹快捷方式，例如自定义添加光影和材质包的文件夹（

**Comment by Chiloven945** [comment]
可以交个issue，就不要交discussion了……

**Comment by WForst-Breeze** [comment, marked as answer]
PCL2 不会支持打开光影、资源包文件夹。您可以使用自定义主页来完成您的需求，如果您迫切希望 PCL2 可以允许用户 **手动添加** 文件夹快捷方式，请提交 新功能提案
